### DISCUSSION TITLE
ERRORS while using the "group_features" in setup stage

### DISCUSSION THREAD

**Discussion by zhangyijue1984** [pycaret/pycaret#4043] [Q&A]
When I set up the "group_features", an errors comes below:
![image](https://github.com/user-attachments/assets/b43deb83-c0ca-4d4e-a89f-1573e3a40ddc)
and my setup is as follow:
![image](https://github.com/user-attachments/assets/71e6cb8c-3247-4188-b198-2bdf8d374139)

whatever in the "group_features" I use the single "[]" or double "[]", the errors were the same.

Then I check my data, the results are:
1. no typo in the list of group_features, because  `df[group_features]` shows the data successfully.
2. the data was already imputed by MissForest, theres is no NaN in it.
3. the data type is "float64"

Please help me to find out what problems I have made.

P.S.  pycaret version is 3.0

**Comment by celestinoxp** [comment]
@zhangyijue1984 can you give more details? the complete error log?
